tests/unit/auth/test_rest.py

```diff
@@ -0,0 +1,43 @@
+import salt.auth.rest
```

Comment:
Would it be asking too much to write these tests for pytest and not unittest under `tests/pytests/unit`?

Reply:
I'll try to do that

Reply:
I'll try to do that

Reply:
Thanks!

Reply:
Done move test to pytest

Reply:
Thank You!